plugins/repository-gcs/build.gradle

```diff
@@ -311,7 +318,10 @@ testClusters {
 }
 
 task gcsThirdPartyTest(type: Test) {
-  dependsOn integTest,largeBlobIntegTest
+  SourceSetContainer sourceSets = project.getExtensions().getByType(SourceSetContainer.class);
```

Comment:
We should think about building up some kind of convenience around this sort of thing.

Reply:
Why isn't the `yaml-rest-test` plugin doing this setup?

Reply:
It's not that simple. By default the `java` plugin wires up all tasks of type `Test` with the `test` source set. What happens when we have a project with multiple test source sets, and then we manually create a `Test` task? Which source set should it default to?

Reply:
Ah sorry, I misread this as the main yaml test, but i realize now this is another test task also utilizing the new source set. I agree then a helper or something to do this would be good, as it seems used several times in this PR.

Reply:
Agreed , I will take a look (in a future PR) for any way to simplify this boiler plate.